indico/modules/events/papers/operations.py

```diff
@@ -130,32 +129,32 @@ def close_cfp(event):
     event.log(EventLogRealm.management, EventLogKind.negative, 'Papers', 'Call for papers closed', session.user)
 
 
-@no_autoflush
-def create_paper_revision(contribution, submitter, files):
-    paper = Paper(contribution=contribution)
+def create_paper_revision(paper, submitter, files):
     revision = PaperRevision(paper=paper, submitter=submitter)
+    paper.last_revision = revision
```

Comment:
If there's later code relying on this to be up to date, you could call

    db.session.expire(revision._contribution, ['_paper_last_revision'])